packages/core/src/modules/routing/services/MediationRecipientService.ts

```diff
@@ -69,6 +69,31 @@ export class MediationRecipientService {
     this.messageSender = messageSender
   }
 
+  public async requestStatus(
+    config: {
+      mediatorId?: string
+      recipientKey?: string
+    } = {}
+  ) {
+    let mediator
+    let mediatorRecord
```

Comment:
I am not a big fan of a `let` like this at the top without a declaration. Is there a way we can rewrite this function to make it a bit cleaner? I can give it a go aswell.

Reply:
This was a result of me not being careful when creating the branch that brought this method into this PR, Timo mentioned below that this was removed because it causes issues with the pick up process. I am removing it from this PR as well